styles/Header.module.css

```
@@ -1,8 +1,333 @@
 
-.link {
-    color: #000000;
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    height: 75px;
+    padding: 0 50px;
+    position: sticky;
+    top: 0;
+    background-color: white;
+    box-shadow: 0px 5px 15px 0 rgba(0, 0, 0, 0.15);
+}
+
+.links {
+    display: inline-block;
+    padding: 10px 15px;
+    text-decoration: none;
+    color: black;
 }
 
 .activeLink {
-    color: #ff0000;
-}
\ No newline at end of file
+    color: #5CA2D5;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+.logo {
+    display: inline-block;
+    height: 65px;
+    width: auto;
+    cursor: pointer;
+}
+
+.navLinks {
+    display: flex;
+    align-self: center;
+    padding-left: 30px;
+    flex: 1;
+}
+
+.searchForm {
+    display: flex;
+    width: 30%;
+    min-width: 18%!important;
+}
+
+.searchInput {
+    height: 30px;
+    margin: auto;
+    width: 80%;
+}
+
+.formButton{
+    align-self: center;
+    flex: 1 1 auto;
+    height: 30px;
+    cursor: pointer;
+    background-image: url("/magnifying-glass.svg");
+    background-position: center;
+    background-repeat: no-repeat;   
+    border-style: solid;
+    border-width: 1px;
+    background-size: 20%;
+}
+
+.menuIcon {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+    margin: 0 50px;
+    cursor: pointer;
+    display: none;
+    background-image: url("/menu.svg");
+    background-color: white;
+    background-size: 70%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-style: none; 
+  }
+
+.searchIcon {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+    margin: 0 50px;
+    font-size: 18px;
+    color:black;
+    cursor: pointer;
+    display: none;
+    background-image: url("/magnifying-glass.svg");
+    background-color: white;
+    background-size: 70%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-style: none; 
+  }
+
+
+.xIcon {
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  margin-top: 0px;
+  padding-left: 40px;
+  background-image: url("/cancel.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  border-style: none;
+  background-color: white;
+  background-size: 50%;
+  cursor: pointer;
+  min-width: 50px;
+  min-height: 50px;
+}
+
+.menuIcon span,
+.cancelIcon,
+.searchIcon {
+    display: none;
+}
+
+.mobileNavMenu {
+  position: fixed;
+  top: 0;
+  width: 200px;
+  height: 100vh;
+  padding: 30px;
```

```suggestion
  padding: 15px;
```

I think this could have a bit less padding